yt_spoti_converter_django_application/urls.py

```diff
@@ -3,4 +3,7 @@
 
 urlpatterns = [
     path("", views.index, name="index"),
-]
\ No newline at end of file
+    path("authentication/", views.authenticaiton, name="authentication"),
+    path("converter/", views.converter, name="converter"),
+    path("end/", views.end, name="end"),
```

Comment:
you should avoid generic imports and import individual items in order not to pollute the namespace and yield funky behaviour when names clash:
`from views import converter`